src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/controller/LimitedPartnerController.java

```diff
@@ -24,11 +28,7 @@
 import java.util.HashMap;
 
 import static uk.gov.companieshouse.api.util.security.EricConstants.ERIC_IDENTITY;
-import static uk.gov.companieshouse.limitedpartnershipsapi.utils.Constants.ERIC_REQUEST_ID_KEY;
-import static uk.gov.companieshouse.limitedpartnershipsapi.utils.Constants.TRANSACTION_KEY;
-import static uk.gov.companieshouse.limitedpartnershipsapi.utils.Constants.URL_GET_LIMITED_PARTNER;
-import static uk.gov.companieshouse.limitedpartnershipsapi.utils.Constants.URL_PARAM_SUBMISSION_ID;
-import static uk.gov.companieshouse.limitedpartnershipsapi.utils.Constants.URL_PARAM_TRANSACTION_ID;
+import static uk.gov.companieshouse.limitedpartnershipsapi.utils.Constants.*;
```

Comment:
need to use single imports
change your intellij settings to do this: https://stackoverflow.com/a/25916603